### DISCUSSION TITLE
utilizing PANDA instead of GRNBoost2

### DISCUSSION THREAD

**Discussion by kh49** [aertslab/SCENIC#254] [* General SCENIC questions]
Is it possible to utilize output from [PANDA](https://sites.google.com/a/channing.harvard.edu/kimberlyglass/tools/panda) for the initial GRN links instead of GENIE3 or GRNBoost2? At first glance, the data formats seem fairly similar, but I'm not clear on whether any conversion is required or not.

**Comment by s-aibar** [comment]
Dear @pophipi,

The input to RcisTarget is a set of co-expressed genes. You can infer those by any other method instead of GENIE3/GRNBoost, and start SCENIC on the "create regulons step" (`runSCENIC_2_createRegulons()`: https://github.com/aertslab/SCENIC/blob/master/vignettes/detailedStep_2_createRegulons.Rmd). Of course, the final network results will be different... but if you prefer to use PANDA for any reason, you can certainly give it a try. 

There is an example of the input format atk: http://scenic.aertslab.org/examples/SCENIC_MouseBrain/int/2.1_tfModules_forMotifEnrichmet.Rds (e.g. co-expression gene sets as a list, including the candidate TF/regulator in the name).
